build.gradle

```diff
@@ -30,8 +29,8 @@ repositories {
 }
 
 dependencies {
-	minecraft "com.mojang:minecraft:1.16.4"
-	mappings "net.fabricmc:yarn:1.16.4+build.9:v2"
+	minecraft "com.mojang:minecraft:1.16.5"
+	mappings "net.fabricmc:yarn:1.16.5+build.6:v2"
```

Comment:
This is really a test dependency. Would be good if we move these mc deps to version specific test subprojects

Reply:
I think so. I'll revert it.

Reply:
Will keep this open for now.